frameworks/mysqldump/export/test

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
```

Comment:
This file should be named something other than `test` since that is a special-name in the base of a framework directory.

Reply:
Whoops, intention was to put it in `src` - good catch